demo-projects/meetup/server.js

```diff
@@ -19,6 +19,11 @@ Promise.all([keystone.prepare({ port }), nextApp.prepare()])
     // Attach the auth routes
     server.app.use('/', createAuthRoutes(keystone));
 
+    server.app.get('/event/:id', (req, res) => {
```

Comment:
Irrelevant after next-routes has been added, but this should really be colocated with the next request handler